easybuild/easyblocks/generic/pythonpackage.py

```diff
@@ -178,17 +178,18 @@ def extra_options(extra_vars=None):
         if extra_vars is None:
             extra_vars = {}
         extra_vars.update({
-            'unpack_sources': [True, "Unpack sources prior to build/install", CUSTOM],
+            'buildcmd': ['build', "Command to pass to setup.py to build the extension", CUSTOM],
+            'download_dep_fail': [False, "Fail if downloaded dependencies are detected", CUSTOM],
```

Comment:
Set to `False` for backward-compatibility, but we should consider switching this to `True` by default for EasyBuild v4.0.0.

Problem is that we'll need to fix quite a bit of easyconfigs...

We could try to enforce a policy where new easyconfigs for Python packages include `download_dep_fail = True`?